pkg/operator/ceph/csi/secrets.go

```diff
@@ -212,9 +259,175 @@ func CreateCSISecrets(context *clusterd.Context, clusterInfo *client.ClusterInfo
 		return errors.Wrap(err, "failed to create kubernetes csi secret")
 	}
 
+	err = updateCephStatusWithCephxStatus(context, clusterInfo, clusterNamespaced, currentKeyCount)
```

Comment:
Let's call `UpdatedCephxStatus()` and pass it into this function rather than having the function calculate it.

We probably want `createCSIKeyring()` to return a bool that says whether or not the key was rotated. We can pass OR'ed value of the 4 values to `UpdatedCephxStatus()`.

Reply:
done